yarn-project/circuits.js/src/structs/avm/avm.ts

```diff
@@ -851,13 +851,13 @@ export class AvmExecutionHints {
   public readonly contractInstances: Vector<AvmContractInstanceHint>;
   public readonly contractBytecodeHints: Vector<AvmContractBytecodeHints>;
 
-  public readonly storageReadRequest: Vector<AvmPublicDataReadTreeHint>;
-  public readonly storageUpdateRequest: Vector<AvmPublicDataWriteTreeHint>;
-  public readonly nullifierReadRequest: Vector<AvmNullifierReadTreeHint>;
-  public readonly nullifierWriteHints: Vector<AvmNullifierWriteTreeHint>;
-  public readonly noteHashReadRequest: Vector<AvmAppendTreeHint>;
-  public readonly noteHashWriteRequest: Vector<AvmAppendTreeHint>;
-  public readonly l1ToL2MessageReadRequest: Vector<AvmAppendTreeHint>;
+  public readonly publicDataReads: Vector<AvmPublicDataReadTreeHint>;
+  public readonly publicDataWrites: Vector<AvmPublicDataWriteTreeHint>;
+  public readonly nullifierReads: Vector<AvmNullifierReadTreeHint>;
+  public readonly nullifierWrites: Vector<AvmNullifierWriteTreeHint>;
+  public readonly noteHashReads: Vector<AvmAppendTreeHint>;
+  public readonly noteHashWrites: Vector<AvmAppendTreeHint>;
+  public readonly l1ToL2MessageReads: Vector<AvmAppendTreeHint>;
 
```

Comment:
Just a rename because "request" doesn't apply anymore since the AVM itself does reads and writes and it doesn't delegate to a later circuit.